bluemira/base/reactor_config.py

```diff
@@ -245,6 +281,13 @@ def _extract(self, arg_keys: Tuple[str], is_config: bool) -> dict:
                 # get the values from the _PARAMETERS_KEY
                 to_extract = current_layer.get(_PARAMETERS_KEY, {})
 
+            if not isinstance(to_extract, dict):
+                raise ReactorConfigError(
+                    f"Arg ${current_arg_key} is too specific, "
```

Comment:
I'm not sure what '`too specfic`' means, here. Probably just me not following the code very well, but in what cases would I expect to see this error? Also, there's no test for this error case

Reply:
Ah okay, good catch.

I thought the below few lines explained it more.
It's saying if you do something like config_for('a', 'b', 'c'), the value of 'c' must be either a path or a dict. If it's a literal value, you're probably doing something wrong.

It would like doing config_for('IVC', 'Wall silhouette', 'run_mode')
which is wrong.
It should be config_for('IVC', 'Wall silhouette')['run_mode'] if you wanted just 'run_mode'